### ISSUE TITLE
Architecture

### ISSUE THREAD

**Issue by moscalej** [moscalej/Deep_learning_hw#20]
Architecture selection is major and challenging task when constructing DNNs.
Research how the architecture selection affects the results for the MNIST dataset.
Find the best architecture from all architectures up to depth of 3 and width of
512 (conduct a grid search of at least 9 different architectures). In your report
discuss how the width, depth and the overhaul number of weights affect the
train and test accuracy. Motivate your selection of architecture from a statistical
learning theory perspective (hypothesis set size, training set size, overfitting
etc...)

┆Attachments: <a href="http:&#x2F;&#x2F;scikit-learn.org&#x2F;stable&#x2F;modules&#x2F;grid_search.html">http:&#x2F;&#x2F;scikit-learn.org&#x2F;stable&#x2F;modules&#x2F;grid_search.html</a> | <a href="https:&#x2F;&#x2F;trello-attachments.s3.amazonaws.com&#x2F;5af73f0cd3ba6ec07ed3668b&#x2F;5aff4dbfdf15f9e956816371&#x2F;5ab52068a42f79825b362564a3c14513&#x2F;image.png">image.png</a>
